frontend/compositions/enrollment-response/enrollment-response.module.css

```diff
@@ -0,0 +1,29 @@
+.response {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: left;
+}
+
+.message {
+  text-align: left;
+  line-height: var(--size24);
+  max-width: var(--size512);
+}
+
+.message p {
```

Comment:
Are we certain that descendant selectors work as expected with CSS modules? I've had trouble with this working in my own tickets, so maybe take this review point with a grain of salt if you're certain that this is working as expected.

Reply:
I've been using descendant selectors just fine if that helps!